tools/ratbag-command.c

```diff
@@ -351,7 +364,8 @@ ratbag_cmd_info(const struct ratbag_cmd *cmd,
 		if (!profile)
 			continue;
 
-		printf("  Profile %d%s\n", i,
+		printf("  Profile %d (%sabled)%s\n", i,
+		       ratbag_profile_is_enabled(profile) ? "en" : "dis",
```

Comment:
urgh, I really don't like that, it makes it impossible to grep for the message for effectively no benefit. just make it ? "enabled" : "disabled"